sip-ri/src/main/java/gov/nist/core/NameValueList.java

```diff
@@ -57,13 +57,17 @@ public class NameValueList implements Serializable, Cloneable, Map<String,NameVa
 
     private Map<String,NameValue> hmap;
 
+    protected boolean isQuotedString; //@MukhaValeriia
+    private String quotes; 	//@MukhaValeriia		 
     private String separator;
     
+    
+    private boolean escaped; //@MukhaValeriia
```

Comment:
This escaped variable doesn't seem to be used. Can you please clean it up if that's the case?